front-end/src/ui-components/AccountSelectionForm.tsx

```diff
@@ -8,18 +8,20 @@ import React from 'react';
 import { DropdownProps } from 'semantic-ui-react/dist/commonjs/modules/Dropdown';
 import { Form } from 'src/ui-components/Form';
 
+import Balance from '../components/Balance';
 import AddressDropdown from './AddressDropdown';
 import HelperTooltip from './HelperTooltip';
 
 interface Props{
 	title: string
-    accounts: InjectedAccountWithMeta[]
-    address: string
-    className?: string;
-    onAccountChange: (event: React.SyntheticEvent<HTMLElement, Event>, data: DropdownProps) => void
+	accounts: InjectedAccountWithMeta[]
+	address: string
+	balance?: boolean
```

Comment:
Let's call it `withBalance`, and set it to `true`, no need to have anything dealing with the api in the props.
```suggestion
	withBalance?: boolean
```